config/locales/models/groups/en.yml

```diff
@@ -11,6 +11,9 @@ en:
           attributes:
             repo_name:
               taken: Repository %{value} already exists.
+              format: must only contain alphanumeric, hyphen, a blank space, or underscore
```

Comment:
There were other options for the key chaining, I thought using the longest one would be better since its the most explicit, but I could also use `errors.attributes.[attribute_name].[key]` instead